kubectl/README.md

```diff
@@ -0,0 +1,51 @@
+# Knative with knctl
```

Comment:
"Knative"? ...copy-paste artifact? should this say "Kubectl" instead?

Reply:
Ah, again